support-frontend/assets/components/subscriptionCheckouts/address/addressFields.jsx

```diff
@@ -134,59 +135,65 @@ class AddressFields<GlobalState> extends Component<PropTypes<GlobalState>> {
             }
           }}
         /> : null}
-        <InputWithError
+        <TextInput
+          css={marginBottom}
           id={`${scope}-lineOne`}
           label="Address Line 1"
           type="text"
           value={props.lineOne}
-          setValue={props.setAddressLineOne}
+          onChange={e => props.setAddressLineOne(e.target.value)}
           error={firstError('lineOne', props.formErrors)}
         />
-        <InputWithError
+        <TextInput
+          css={marginBottom}
           id={`${scope}-lineTwo`}
           label="Address Line 2"
           optional
           type="text"
           value={props.lineTwo}
-          setValue={props.setAddressLineTwo}
+          onChange={e => props.setAddressLineTwo(e.target.value)}
           error={firstError('lineTwo', props.formErrors)}
         />
-        <InputWithError
+        <TextInput
+          css={marginBottom}
           id={`${scope}-city`}
           label="Town/City"
           type="text"
           maxlength={40}
           value={props.city}
-          setValue={props.setTownCity}
+          onChange={e => props.setTownCity(e.target.value)}
           error={firstError('city', props.formErrors)}
         />
         <MaybeSelect
+          css={marginBottom}
           id={`${scope}-stateProvince`}
           label={props.country === 'CA' ? 'Province/Territory' : 'State'}
           value={props.state}
-          setValue={props.setState}
+          onChange={e => props.setState(e.target.value)}
           error={firstError('state', props.formErrors)}
           isShown={AddressFields.shouldShowStateDropdown(props.country)}
         >
-          <option value="">--</option>
+          <option value="">{`Select a ${props.country === 'CA' ? 'province/territory' : 'state'}`}</option>
           {AddressFields.statesForCountry(props.country)}
         </MaybeSelect>
         <MaybeInput
+          css={marginBottom}
           id={`${scope}-stateProvince`}
           label="State"
           value={props.state}
-          setValue={props.setState}
+          onChange={props.setState}
```

Comment:
Regarding replacing `setValue` with `onChange` here: The `setValue` prop for this input  originally took `props.setState`, I'm assuming `setValue` pulled `e.target.value` and passed this onto `props.setState`. Now we're using `onChange` we'll be passing the `Event` (`e`) as the argument to `props.setState`. If `props.setState` is actually expecting `e.target.value` then should we not update this to be `onChange={e => props.setState(e.target.value)}`, as you've done for the `<TextInput>` below?

Reply:
Yes, that's right. Thanks for picking this up.